examples/scripts/lora_example.py

```
@@ -0,0 +1,370 @@
+    return parser.parse_args()
+
+
+def main():
+    """Main training function."""
+    args = parse_arguments()
+
+    # Handle sample data creation
+    if args.create_sample_data:
+        sample_path = "./sample_data.jsonl"
+        create_sample_data(sample_path, args.sample_data_format)
+        print(f"\\nUse this sample data with:")
```

Ruff (0.14.5)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)